script/build-icons

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+dir="$(cd "$(dirname "$0")" && cd "../src/common/icons" && pwd)"
+dark_src="${dir}/icon.svg"
+light_src="${dir}/icon-light.svg"
+
+export_png_icon() {
+  src=$1 dest=$2
+  inkscape -z -o "$dest" -D -w "$s" "$src" # export
+  convert "$dest" -background none -gravity center -extent "${s}x${s}" "$dest" # pad
+  pngcrush -brute -ow "$dest" # reduce
+}
```

Comment:
We should probably pass the target size as a third argument. The variable `s` is only defined below and using it here introduces a very indirect dependency.

```suggestion
function export_png_icon() {
  local src=$1
  local dest=$2
  local size=$3

  inkscape -z -o "$dest" -D -w "$size" "$src" # export
  convert "$dest" -background none -gravity center -extent "${size}x${size}" "$dest" # pad
  pngcrush -brute -ow "$dest" # reduce
}
```

Reply:
Very very good catch 👏 

Added it ✅ 

I skipped `function` and `local` though as I kept using `sh` and these super advanced features aren't available there 😉 (they are also not super critical here).